src/components/AddPlaylist/AddPlaylist.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ChangeEvent, FormEvent, useState } from "react";
+import { useDispatch } from "react-redux";
+import { playlistsAsyncActions } from "../../store/slices/playlists/slice";
+import { Button, Container, FormContainer, HelperText, StyledInput, Title, Wrapper } from "./styles";
+
+const AddPlaylist = () => {
+    const [name, setName] = useState<string>("");
+    const [helperText, setHelperText] = useState<string | null>(null);
+    const dispatch = useDispatch();
+
+    const savePlaylistName = (event: ChangeEvent<HTMLInputElement>) => {
+        let name = event.target.value;
+        if (name.length === 0) {
+            setHelperText("Cannot be empty!");
+            setName("");
+        } else if (name.length > 50) {
+            setName(name);
+            setHelperText("Too long name for the playlist!");
+        } else {
+            setName(name);
+            setHelperText(null);
+        }
+    };
+
+    const createPlaylist = (event: FormEvent<HTMLButtonElement>) => {
+        event.preventDefault();
+        if (helperText) {
+            return;
+        }
+        dispatch(playlistsAsyncActions.createPlaylist(name));
+        setName("")
+    };
+
+    return (
+        <Wrapper>
+            <Container>
+                <Title>Create a new playlist</Title>
+                <FormContainer>
+                    <StyledInput
+                        type="text"
```

Comment:
instead of the whole savePlaylistName function, you could use the html input min/maxlength attribute.

Reply:
This way it could be also helpful for the validation of the dispatch, when we check that the helperText cannot be null.